forms-html/AhmadA.Duhoki-ZainabSaud/index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>2nd Day of HTML Test</title>
+</head>
+<body>
+    <form method="POST">
+        <label for="email">Enter Your Email:</label>
+            <br>
+        <input type="email" required name="email" id="email">
+            <br>
+        <label for="username">Username:</label>
+            <br>
+        <input type="text" name="username" id="username">
+            <br>
+        <label for="password">Password:</label>
+            <br>
+        <input type="password" name="password" id="password">
+            <br>
+        <label for="quantity">Quantity(between 1 and 5):</label>
+        <input type="number" max="5" min="1" name="quantity" id="quantity">
+            <br>
+        <label for="volume">Volume(between 0 and 50):</label>
+        <input type="range" max="50" min="0" name="volume" id="volume">
+            <br>
+            <br>
+        <P>radio</P>
+            <br>
+        <input type="radio" name="gender" id="male" value="Male">
+                <label for="male">Male</label>
+            <br>
+        <input type="radio" name="gender" id="female" value="Female">
+            <label for="female">Female</label>
+                <br>
+        <input type="radio" name="gender" id="other" value="other">
+            <label for="other">Other</label>
```

Comment:
Good job on using the same name for the radio buttons. This ensures that only one radio button gets selected. 👍